writer.go

```diff
@@ -275,6 +289,13 @@ func (w *Writer) WriteMessages(ctx context.Context, msgs ...Message) error {
 		}
 
 		for _, msg := range msgs {
+			if int(msg.message().size()) > w.config.MaxMessageBytes {
+				err := errors.New(fmt.Sprintf("The message is %d bytes "+
```

Comment:
Any reasons not to use `fmt.Errorf` here?

Reply:
I don't think so, I'll update it.